.eslintrc.js

```diff
@@ -13,6 +13,11 @@ module.exports = {
     'jsx-a11y/click-events-have-key-events': 'off',
     'jsx-a11y/no-static-element-interactions': 'off',
     'jsx-a11y/anchor-is-valid': 'off',
+    '@typescript-eslint/no-use-before-define': 'off',
+    'class-methods-use-this': 'off',
+    'max-classes-per-file': 'off',
+    'no-case-declarations': 'warn',
```

Comment:
Ogólnie to nie trzeba tego wyłączać, tylko trzeba wiedzieć jak dobrze napisać w takim przypadku `case/default` dla `switcha`. Zamiast deklarować `case` w taki sposób:
```
    case 'value':
        let temp = 1
        break
```
to zamknąć to w klamry:
```
    case 'value': {
        let temp = 1
        break
    }
```
wtedy problem się rozwiąże sam, bez usuwania tej reguły :D